lib/oeqa/selftest/updater.py

```diff
@@ -31,23 +31,13 @@ def test_deploy_help(self):
         result = runCmd('%s --help' % p, ignore_status=True)
         self.assertEqual(result.status, 0, "Status not equal to 0. output: %s" % result.output)
 
-
-class GarageSignTests(oeSelfTest):
-
-    @classmethod
-    def setUpClass(cls):
-        logger = logging.getLogger("selftest")
-        logger.info('Running bitbake to build garage-sign-native')
-        bitbake('garage-sign-native')
-
-    def test_help(self):
+    def test_garagesign_help(self):
         bb_vars = get_bb_vars(['SYSROOT_DESTDIR', 'bindir'], 'garage-sign-native')
```

Comment:
Looks like we missed this one: instead of `garage-sign-native` this should be `aktualizr-native`.